src/elements/content-sidebar/flowTypes.js

```diff
@@ -29,6 +30,7 @@ type AdditionalSidebarTab = {
     iconUrl?: string,
     id: number,
     title: ?string,
+    icon?: IconButton,
```

Comment:
this should be an svg type, not a button right? otherwise it would be a blueprint iconbutton inside of a buie plainbutton